### DISCUSSION TITLE
can you supply a gz format of aarch64 clickhouse

### DISCUSSION THREAD

**Discussion by l1t1** [ClickHouse/ClickHouse#48807] [Q&A] [answered]
I use `curl https://clickhouse.com/ | sh`
It cannot resume and often abort.

**Comment by rschu1ze** [comment, marked as answer]
Kindly see [here](https://clickhouse.com/docs/en/install#from-tgz-archives) and [here](https://packages.clickhouse.com/tgz/).

**Comment by l1t1** [comment]
thanks
